sqlalchemy_ibmi/base.py

```diff
@@ -813,7 +804,7 @@ def _get_default_schema_name(self, connection):
 
     @property
     def default_schema_name(self):
-        return self.dialect.default_schema_name
+        return self._get_default_schema_name()
```

Comment:
_get_default_schema_name takes a connection object, but we're not passing it. Probably need to pass `self.connection`.